pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -83,11 +91,22 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 
 	total := api.EmptyResource()
 	used := api.EmptyResource()
+	lockedNodesIdle := api.EmptyResource()
+	if util.Reservation.TargetJob != nil && len(util.Reservation.LockedNodes) != 0 {
+		for _, node := range util.Reservation.LockedNodes {
+			lockedNodesIdle.Add(node.Idle)
+			klog.V(3).Infof("locked node: %s, idle CPU: %f", node.Name, node.Idle)
```

Comment:
why only print cpu resource in this file?

Reply:
This log will be removed because i take cpu as test metric when self test